cli/cmd/inject.go

```diff
@@ -439,6 +446,9 @@ func (options *proxyConfigOptions) overrideConfigs(configs *cfg.All, overrideAnn
 	if options.traceCollectorSvcAccount != "" {
 		overrideAnnotations[k8s.ProxyTraceCollectorSvcAccountAnnotation] = options.traceCollectorSvcAccount
 	}
+	if options.waitBeforeExit != 0 {
+		overrideAnnotations[k8s.ProxyWaitBeforeExitAnnotation] = time.DurationToSecondsString(options.waitBeforeExit) + "s"
```

Comment:
Instead of introducing a new dependency on `github.com/docker/docker/api/types/time`, you could do `options.waitBeforeExit.Seconds()`

Reply:
s/`Seconds()`/`String()`